### PR TITLE
Feature/mhip 323

### PR DESCRIPTION
**Pull request by meyerar**
This feature branch deals with MHIP 308, 309, 319, 322, 323
Code refactoring
Structured logging in Lambda
Database provisioning
Database migration - see attachment
Lambda invocation from github workflow - kick of database migration from github workflow
FHIR Projection - see attachment
mapping
migration

### CODE REVIEW

**Review by somithajosephHSL — APPROVED** [review]

File: .github/workflows/dev.yml [hunk 1]
```diff
@@ -12,6 +12,7 @@ env:
   AWS_S3_BUCKET: patient-referral-service-dev
   AWS_REGION: ap-southeast-2
   AWS_STACK_NAME: Referrals-Query
+  ALERTS_STACK_NAME: Patient-Referrals-Alerts-sandbox
```
Comment (wliuhealthnz) [inline .github/workflows/dev.yml:15]: Are you re-using Patient-Referrals-Alerts-sandbox in dev?

**Comment by wliuhealthnz** [comment]
can we use formal name "HealthShare" instead of 'HS' for company name?  To keep it consistent with existing projects in bitbucket

**Review by wliuhealthnz — APPROVED** [review]
Looking good

**Comment by meyerar** [comment]
will setup separate discussion on code changes
will need to update develop ci pipelines

**Pull request closed without merging**